.env

```diff
@@ -1,4 +1,9 @@
-# On running the 'build' commands this file is replaced by the files in `env-config/`
-# To ignore and keep this file we have a command in `postshrinkwrap` which assumes this file is not changing
-# The command is `git update-index --assume-unchanged .env`
-# Source: https://wildlyinaccurate.com/git-ignore-changes-in-already-tracked-files/
\ No newline at end of file
+## NB only variables prefixed with SIMORGH_ are available on both the server and client
+SIMORGH_BASE_URL=http://localhost:7080
+SIMORGH_PUBLIC_STATIC_ASSETS_ORIGIN=http://localhost:7080
+SIMORGH_PUBLIC_STATIC_ASSETS_PATH=/
+SIMORGH_ATI_BASE_URL=https://logws1363.ati-host.net?
+SIMORGH_MPULSE_API_KEY=JLSEX-R76DK-JXL3A-UB7YH-NPSEW
+LOG_DIR='log'
+LOG_LEVEL='debug'
+SIMORGH_APP_ENV=local
```

Comment:
Oops

Reply:
omg

Reply:
Fixed. Looking forward to .env's reappearance in my next PR... 🤦‍♂